Makefile

```diff
@@ -8,13 +8,13 @@ DIST_DIRS         = find * -type d -exec
 ifdef DEBUG
 GOFLAGS   := -gcflags="-N -l"
 else
-GOFLAGS   := 
+GOFLAGS   :=
 endif
 
 # go option
 GO        ?= go
 TAGS      :=
-LDFLAGS   := 
+LDFLAGS   :=
```

Comment:
I think that space is there for a reason.

Reply:
Editor stripped it...will revert

Reply:
ignore these, I'm no longer bothered by these phantoms